src/components/SigningModals/TransactionJourney.js

```diff
@@ -0,0 +1,265 @@
+          border-radius: 100%;
+
+          top: 0;
+          left: 0;
+          right: 0;
+          bottom: 0;
+        }
+
+        &:after {
+          animation: ${pulseAnimation} 2s ease infinite;
+
+          top: -${size}px;
+          bottom: -${size}px;
+          left: -${size}px;
+          right: -${size}px;
+
+          opacity: 0.2;
+        }
+      `}
+    />
+  )
+}
+
+function ProgressLine({ status, pulse, themeColor }) {
+  const theme = useTheme()
+
+  const pipSize = 1 * GU
+  const pipOffset = ITEM_SPACING + 1.25 * GU
+
+  const highlight = css`
+    &::before {
+      content: '';
+
+      bottom: ${pulse && status === CURRENT ? pipOffset : 0}px;
+      border-left: 1px solid ${themeColor};
+
+      z-index: 1;
+    }
+  `
+
+  return (
+    <div
+      css={`
+        position: absolute;
+
+        bottom: 0px;
+        top: 0;
+        left: 50%;
+
+        &::after,
+        &::before {
+          position: absolute;
+
+          top: 0;
+          left: 0;
+        }
+
+        &::after {
+          content: '';
+          bottom: 0;
+          border-left: 1px dashed ${theme.surfaceOpened};
+        }
+
+        ${status !== NEXT && highlight}
+      `}
+    >
+      {status === CURRENT && pulse && (
+        <PulsePip
+          size={pipSize}
+          themeColor={themeColor}
+          css={`
+            position: absolute;
+
+            left: -${pipSize / 2}px;
+            bottom: ${pipOffset}px;
+
+            z-index: 5;
+          `}
+        />
+      )}
+    </div>
+  )
+}
+
+function JourneyItem({
+  icon,
+  title,
+  children,
+  status,
+  isLastItem,
+  pulse,
+  themeColor,
+}) {
+  const theme = useTheme()
+
+  return (
+    <li
+      css={`
+        display: flex;
+      `}
+    >
+      <div
+        css={`
+          position: relative;
+          margin-right: ${2.5 * GU}px;
+        `}
+      >
+        {!isLastItem && (
+          <ProgressLine status={status} pulse={pulse} themeColor={themeColor} />
+        )}
+        <div
+          css={`
+            position: relative;
+            display: flex;
+
+            overflow: hidden;
+
+            width: ${5 * GU}px;
+            height: ${5 * GU}px;
+
+            background-color: ${theme.badge};
+            border-radius: ${RADIUS}px;
+            z-index: 1;
+
+            &::after {
+              content: '';
+
+              background-image: url(${icon});
+              flex: 1;
+
+              ${status === NEXT &&
+                `filter: grayscale(97%);
+                mix-blend-mode: hard-light;
+                opacity: 0.8;
+              `}
+            }
+          `}
+        />
+      </div>
+      <div
+        css={`
+          ${!isLastItem && `padding-bottom: ${ITEM_SPACING}px;`}
+        `}
+      >
+        <h2
+          css={`
+            ${textStyle('body1')};
+
+            margin-bottom: ${0.5 * GU}px;
+
+            color: ${status === NEXT ? theme.contentSecondary : theme.content};
```

Comment:
Just wondering, but could we lift this to the upper `div` to apply it to all the content areas (title + children)?

Reply:
Totally